hack/allocate.sh

```diff
@@ -22,33 +22,35 @@ set -o pipefail
 
 main() {
 
-  local serving_version=v0.22.0
-  local eventing_version=v0.22.0
-  local kourier_version=v0.22.0
+  local kubernetes_version=v1.21.1
+  local knative_version=v0.23.0
+  local kourier_version=v0.23.0
 
   local em=$(tput bold)$(tput setaf 2)
```

Comment:
this doesn't seem to work in CI runner:
`tput: No value for $TERM and no -T specified`

Reply:
Thanks, defaulted TERM to 'dumb' for CI which has none set.

Reply:
Wouldn't `ansi` work too? github action do support ansi escape codes.